pkg/services/policer/process.go

```diff
@@ -2,113 +2,80 @@ package policer
 
 import (
 	"context"
-	"sync"
+	"errors"
+	"time"
 
+	"github.com/nspcc-dev/neofs-node/pkg/local_object_storage/engine"
+	"github.com/nspcc-dev/neofs-sdk-go/object"
 	"go.uber.org/zap"
 )
 
-// Task represents group of Policer tact parameters.
-type Task struct{}
-
-type prevTask struct {
-	undone int
-
-	cancel context.CancelFunc
-
-	wait *sync.WaitGroup
-}
-
-type workScope struct {
-	val int
-
-	expRate int // in %
-}
-
 func (p *Policer) Run(ctx context.Context) {
-	defer func() {
```

Comment:
Were these logs so useless?

Reply:
Nope, accidentally removed it. Put it back.